samples/msal-react-samples/default/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Unsure if the React sample needs a specific readme on how to use it?

Reply:
Yes! Thanks for calling this out, forgot about this

Reply:
Done!